client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -67,8 +81,19 @@ export class PageViewTracker extends React.Component {
 	}
 }
 
+const mapStateToProps = state => {
+	const selectedSiteId = getSelectedSiteId( state );
+	const selectedSiteSlug = getSiteSlug( state, selectedSiteId );
+	const currentSlug = getSiteFragment( get( window, 'location.pathname', '' ) );
```

Comment:
We might also need to do a check here to see if the fragment is a number. We do also accept the siteId directly.